samples/scala-protobuf-eventsourced-customer-registry-subscriber/docker-compose.yml

```diff
@@ -1,17 +1,17 @@
 version: "3"
 services:
   # note the ports being different from other sample docker-compose files to allow this service to run
-  # on the same local machine as the scala-customer-registry-
+  # on the same local machine as the scala-protobuf-eventsourced-customer-registry
   kalix-proxy:
-    image: gcr.io/kalix-public/kalix-proxy:1.1.8
+    image: gcr.io/kalix-public/kalix-proxy:1.1.9-M2
     ports:
-      - "9001:9001"
```

Comment:
what about calls between components?

Reply:
This is not done in this sample, so I thought that keeping the docker-compose changes to a minimum was better. 

In the documentation we will need to mention that it might be necessary to change that port as well.